sorc/ncep_post.fd/CALVOR.f

```diff
@@ -329,8 +331,16 @@ SUBROUTINE CALVOR(UWND,VWND,ABSV)
 
         call poleavg(IM,JM,JSTA,JEND,SMALL,COSL(1,jsta),SPVAL,ABSV(1,jsta))
         deallocate (wrk1, wrk2, wrk3, cosl, iw, ie)
-     
-      ELSE IF(GRIDTYPE == 'A')THEN
+
+      ELSE
```

Comment:
@JesseMeng-NOAA Can you add comment as:
ELSE   !modelname is not GFS

Reply:
Done